mediathread/taxonomy/models.py

```diff
@@ -21,12 +21,11 @@ class Vocabulary(models.Model):
     display_name = models.CharField(max_length=50)
     description = models.TextField(null=True, blank=True)
     single_select = models.BooleanField(default=False)
-
+    onomy_url = models.TextField(null=True, blank=True)
```

Comment:
I'm not seeing the schema migration file for this field addition. Can you verify it has been added to this pull request?